api/src/backend/api/models.py

```diff
@@ -251,6 +251,21 @@ def validate_m365_uid(value):
                 pointer="/data/attributes/uid",
             )
 
+    @staticmethod
+    def validate_m365_certificate_content(value):
```

Comment:
If this is not explicitly used by model validation, this might belong to the serializer?